types/module/module.go

```diff
@@ -53,7 +52,6 @@ type AppModuleBasic interface {
 	ValidateGenesis(codec.JSONCodec, client.TxEncodingConfig, json.RawMessage) error
 
 	// client functionality
-	RegisterRESTRoutes(client.Context, *mux.Router)
```

Comment:
We have 2 choices here:
1. Remove this method.
2. Keep this method for now, remove it during the app wiring refactor. We just provide no-op implementations in the SDK, but maybe if some chains still have custom REST endpoints they can still keep them.

It seems like removing everything (1) is cleaner, but these days I'm a bit more conservative on API breaking changes

Reply:
Option 2 sounds good.